build.gradle

```diff
@@ -280,6 +280,11 @@ task maybeCloneAndBuildDependencies(type: Exec) {
         // the repository does not exist when building checker.jar.
     }
     executable 'checker/bin-devel/build.sh'
+    doLast{
```

Comment:
Nit: I would add a space before `{`.

Reply:
I moved this change (and added some more) to https://github.com/typetools/checker-framework/pull/4473.